### PR TITLE
åsa sieurin project business-site

### PR DESCRIPTION
**Pull request by sieurin**
A lot of compromises in terms of design, due to the fact that I'm not yet a full-fledged frontend developer :) But I'm happy with how everything turned out anyway and I really appreciate working with html & css. Can't wait to learn more!

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,551 @@
...
+  li{
+    font-size: 1.5rem;
+    margin:0.5rem;
+  }
+
+}
+
+/* On screens that are 992px or more (desktop) */
+
+@media screen and (min-width: 992px) {
```
Comment (rawisou) [inline code/style.css:549]: Correct me if I'm wrong. It looks like this we don't need this part as the tablet and desktop view have the same layout?

File: code/script.js [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
+// Selects a "section", and calls a function which will be executed when the element is clicked.
 document.getElementById("section1").onclick = toggle
+document.getElementById("section2").onclick = toggle
+document.getElementById("section3").onclick = toggle
+document.getElementById("section4").onclick = toggle
+
+// Prevent the information from disappearing.
+document.getElementById('form').onsubmit = event => {
+  //event.preventDefault()
```
Comment (rawisou) [inline code/script.js:16]: I'm not sure if you wanted to keep as the code inside the function is commented out? Nice touch to the form by the way! :) I should definitely learn more about this or rewatch the recorded live one more time to get a better understanding. Thank you!

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,551 @@
...
+  font-size: 10px;
+}
+
+.project-title:hover {
+  background-color: black;
+  color: #EED2D5;
+  transition: all 1.5s ease;
+}
+
+.project-title::before {
```
Comment (rawisou) [inline code/style.css:220]: Nice use of pseudo elements! I did not know that we could add "+ " and "-" in the content! Very smart!  Instead, I created 2 divs to make a cross and a minus sign haha! I will definitely apply this technique next time.

**Review by rawisou — COMMENTED** [review]
Hi Åsa,

Your site works as intended in all screen sizes and different browsers. Your project has fulfilled all the **blue** and **red** requirements.
Your code is well organized with extra nice touch of comment sections in CSS. The naming of classes and functions are easy to understand and follow along.
I could not spot any duplicates. Just one small empty media query for desktop and a function that has a commented code inside. Great job!

Overall, I really enjoy learning from your work. I can see that you have put a lot of efforts into the design and functionality. I am inspired by your work ethics!

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,551 @@
...
+  content: "+";
+  position: absolute;
+  right: 7rem;
+}
+
+.project-title.active::before {
+  content: "-";
```
Comment (rawisou) [inline code/style.css:227]: Nice use of pseudo elements! I did not know that we could add "+ " and "-" in the content! Very smart!  Instead, I created 2 divs to make a cross and a minus sign haha! I will definitely apply this technique next time. Thank you very much!

File: code/style.css (hunk 1 above)
Comment (sieurin) [reply code/style.css:549]: ohh. no I just forgot to delete it when I realized I didn't want anything specific for desktop :)

File: code/script.js (hunk 2 above)
Comment (sieurin) [reply code/script.js:16]: Oh it wouldn't be "out-commented". But no, it wasn't necessary for the function. More that it doesn't reload again but "keep" the information from the function..

File: code/style.css (hunk 4 above)
Comment (sieurin) [reply code/style.css:227]: Yes, very code-efficient ;)

**Pull request closed without merging**